src/clarity-angular/popover/dropdown/dropdown.ts

```diff
@@ -16,26 +16,28 @@ import {
 
 import { Point, PopoverOptions } from "../common/popover";
 import { menuPositions } from "./menu-positions";
+import {IfOpenService} from "../../utils/conditional/if-open.service";
 
 // TODO: the ng-content inside the ng-template should ideally just be
 // <ng-content select="clr-dropdown-menu"></ng-content>. Remove .dropdown-menu in 1.0?
 @Component({
     selector: "clr-dropdown",
     template: `
         <ng-content select="[clrDropdownToggle]"></ng-content>
-        <ng-template [(clrPopover)]="open" [clrPopoverAnchor]="anchor" [clrPopoverAnchorPoint]="anchorPoint"
-                     [clrPopoverPopoverPoint]="popoverPoint" [clrPopoverOptions]="popoverOptions">
+        <div class="dropdown-menu-wrapper" [clrPopoverAnchor]="anchor" [clrPopoverAnchorPoint]="anchorPoint"
```

Comment:
This is one of the things I'd like to revisit in a little while once Angular has more features. Right now an extra wrapper is the only way for us to use the Popover directive, but in the future we might be able to dynamically add it to the host of the DropdownMenu component.

Just a note for later.